lib/server.js

```diff
@@ -18,14 +18,18 @@ function createServer(port) {
     response.end(JSON.stringify(body));
   });
 
-  server.listen(port, (err) => {
-    if (err) {
-      console.error(err);
-      process.exit(1);
-      return;
+  function onError(err) {
+    if(err.code == 'EADDRINUSE') {
+      console.log(`${host}:${port} is already in use, please try another port...`);
+    } else {
+      console.log(err);
```

Comment:
Let's use `console.error` here instead of `console.log` so the error gets written to `STDERR` instead of `STDOUT`.

The same should be done for the above error message.